website/docs/r/elasticsearch_domain.html.markdown

```diff
@@ -252,8 +252,11 @@ The **advanced_security_options** block supports the following attributes:
 
 **domain_endpoint_options** supports the following attributes:
 
-* `enforce_https` - (Required) Whether or not to require HTTPS
+* `enforce_https` - (Optional) Whether or not to require HTTPS
```

Comment:
Since this attribute has a default value, we should call it out here. 👍 

```suggestion
* `enforce_https` - (Optional) Whether or not to require HTTPS. Defaults to `true`.
```